opencga-app/app/scripts/azure/arm/azuredeploy.parameters.json

```diff
@@ -3,64 +3,67 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "_artifactsLocation": {
-            "value": ""
+            "value": "https://raw.githubusercontent.com/opencb/opencga/azure/opencga-app/app/scripts/azure/arm/"
         },
         "_artifactsLocationSasToken": {
-            "value": ""
-        },
-        "HDInsightStorageOption": {
-            "value": "DataLake"
+            "value": "?125"

```

Comment:
any reason we pick 125 instead of keeping it empty?

Reply:
No...you're right...this is a copy/paste issue and I should remove it (and the value above it). Thanks!